Week1/homework/js-exercises/randomDog.js

```diff
@@ -0,0 +1,49 @@
+const url = "https://dog.ceo/api/breeds/image/random";
+const xmlBtn = document.querySelector('#xml');
+const axiosBtn = document.querySelector('#axios');
+const dog = document.querySelector('#dog-list');
```

Comment:
I would suggest to call this `dogs` instead, it's a list of dogs and not a one dog, don't you agree?
```suggestion
const dogs = document.querySelector('#dog-list');
```

you need to make few changes in the file afterwords